sopel/modules/xkcd.py

```diff
@@ -25,7 +25,8 @@
     'wiki.xkcd.com',
     'what-if.xkcd.com',
 ]
-sites_query = ' site:xkcd.com -site:' + ' -site:'.join(ignored_sites)
+sites_query = ' site:xkcd.com ' + ''.join('-site:{}'.format(ignored_site)
```

Comment:
Pretty sure this won't work as it will join to form `-site:a-site:b-site:c` without any spaces.